node/core/pvf/prepare-worker/src/lib.rs

```diff
@@ -151,11 +155,23 @@ pub fn worker_entrypoint(socket_path: &str, node_version: Option<&str>) {
 			let prepare_thread = thread::spawn_worker_thread(
 				"prepare thread",
 				move || {
-					let result = prepare_artifact(pvf, cpu_time_start);
+					#[allow(unused_mut)]
```

Comment:
Why is this necessary? (allow(unused_mut))?

Reply:
Without it, the following line on Linux results in a warning about it being unused:

```rs
#[cfg(target_os = "linux")]
let mut result = result.map(|(artifact, elapsed)| (artifact, elapsed, get_max_rss_thread()));
```